src/Paraunit/Configuration/PHPUnitConfig.php

```diff
@@ -64,7 +65,13 @@ private function getConfigFileRealpath(string $inputPathOrFileName): string
         }
 
         if (is_dir($configFile)) {
+            $configFileFallback = $configFile. DIRECTORY_SEPARATOR . self::FALLBACK_CONFIG_FILE_NAME;
```

Comment:
I would move this inside the following `if`, without using an intermediate variable.

Reply:
ci green and some refactor